src/compiler/codegen/codegen.cc

```diff
@@ -92,6 +94,24 @@ namespace verona::compiler
     return std::make_pair(class_item, method_item);
   }
 
+  /**
+   * Writes the verona bytes, literally writes VERONA_NUMBEr
+   * @param code Generator to gen from
+   */
+  void write_verona_bytes(Generator& code)
+  {
+    code.u32(bytecode::VERONA_NUMBER);
+  }
+
+  /**
+   * Writes the verona integers to the bytecode
```

Comment:
"Verona integers" is vague. "magic number" would be better.
Also `write_magic_bytes` and `write_verona_bytes` can be one function (eg. `write_magic_number`).

Reply:
I kept write_verona_bytes because we might want to add more later, but okay sure